packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -761,6 +761,59 @@ class EditableText extends StatefulWidget {
   /// To be notified of all changes to the TextField's text, cursor,
   /// and selection, one can add a listener to its [controller] with
   /// [TextEditingController.addListener].
+  ///
+  /// {@tool dartpad --template=stateful_widget_material}
+  ///
+  /// This example shows how onChanged could be used to check the TextField's
+  /// current value each time the user inserts or deletes a character.
+  ///
+  /// ```dart
+  /// TextEditingController _controller;
+  ///
+  /// void initState() {
+  ///   _controller = TextEditingController();
```

Comment:
First call super.initState() and in dispose() call super.dispose() afterwards. This is boilerplate and docs like this will make it so :-)

Reply:
Good call!